data/category/base/data.json

```diff
@@ -1,25 +1,52 @@
 {
-    "data": {
-        "category-dairy": {
-            "name": "Dairy"
-        },
-        "category-baking" : {
-            "name": "Baking"
-        },
-        "category-produce" : {
-            "name": "Produce"
-        },
-        "category-bakery" : {
-            "name": "Bakery"
-        },
-        "category-deli" : {
-            "name": "Deli"
-        },
-        "category-frozen" : {
-            "name": "Frozen"
-        },
-        "category-canned" : {
-            "name": "Canned"
-        }
+  "data": {
+    "category-dairy": {
+      "name": "Dairy"
+    },
+    "category-baking": {
+      "name": "Baking"
+    },
+    "category-produce": {
+      "name": "Produce"
+    },
+    "category-bakery": {
+      "name": "Bakery"
+    },
+    "category-deli": {
+      "name": "Deli"
+    },
+    "category-frozen": {
+      "name": "Frozen"
+    },
+    "category-canned": {
+      "name": "Canned"
+    },
+    "category-fish": {
+      "name": "Fish"
+    },
+    "category-meat": {
+      "name": "Meat"
+    },
+    "category-herbs-spices": {
+      "name": "Herbs and Spices"
+    },
+    "category-bread-cereal": {
```

Comment:
In my region, no store has bread and cereal near each other.

Reply:
In Germany neither. Bread would be bakery, cereals can be close but they are often also close to baking goods such as flour or sugar

Reply:
I've split them.